go/consensus/tendermint/roothash/roothash.go

```diff
@@ -342,7 +342,11 @@ func (sc *serviceClient) reindexBlocks(currentHeight int64, bh api.BlockHistory)
 						)
 						return fmt.Errorf("failed to unmarshal finalized event: %w", err)
 					}
-
+					tr := sc.trackedRuntime[value.ID]
```

Comment:
`tr` doesn't seem to be used, why not have it inline in the `if` statement? Also below.